cli/commands/clean.js

```diff
@@ -89,6 +89,11 @@ exports.run = function (logger, config, cli) {
 	if (cli.argv.platforms) {
 		async.series(cli.argv.platforms.map(function (platform) {
 			return function (next) {
+				// scan platform SDK specific clean hooks
```

Comment:
Yeah, I just wanted to make it flexible for platform SDK to choose hook directory but I might be thinking too much.